src/typescript/views/todoView.ts

```diff
@@ -24,11 +22,42 @@ class TodoView {
         });
     };
 
+    /**
+     * Render a todo in todo list.
+     * @param {Todo} todo
+     */
     renderTodo = (todo: Todo): void => {
         const todoTemplate = TodoTemplate.createTodo(todo);
 
         this.todoListEl.innerHTML += todoTemplate;
     };
+
+    /**
+     * Add event listener searching todos to the search input.
+     */
+    addEventSearch = (): void => {
+        const inputSearchEl: HTMLInputElement = document.querySelector('.input-search')!;
+        const btnSearchEl: HTMLElement = document.querySelector('.btn-search')!;
+
+        btnSearchEl?.addEventListener('click', async (e: MouseEvent) => {
+            e.preventDefault();
+
+            const value: string = inputSearchEl.value;
+            const nameTodos: NodeList = document.querySelectorAll('.todo-info-name');
+
+            nameTodos.forEach((nameTodo) => {
+                const todoSearchEl = nameTodo.parentNode?.parentNode as HTMLElement;
+                const nameTodoText = nameTodo.textContent?.toLocaleLowerCase() as string;
```

Comment:
I think toLocaleLowerCase is not correct function we should use? COuld you explain if i'm wrong

Reply:
I fixed @phuocduong-agilityio . Please help me check. Thanks!